mobile/rn-cli.config.js

```diff
@@ -10,7 +10,13 @@ module.exports = {
       'node_modules/react-native-svg'
     ),
     mobx: path.resolve(__dirname, 'node_modules/mobx'),
+    'mobx-react': path.resolve(__dirname, 'node_modules/mobx-react'),
```

Comment:
According to [this](https://gist.github.com/parshap/e3063d9bf6058041b34b26b7166fd6bd#global-environment) gist, if we're using the `rn-cli.config.js` solution to include node modules in react native, we'll also need to set the React Native global environment because "node-style" requires expect the global environment to be in a specific state.

Reply:
See comment above...